README.md

```diff
@@ -29,6 +30,12 @@
 - [Terminalizer](https://terminalizer.com/) - Record your terminal and generate animated gif images or share a web player
 - [ttygif](https://github.com/icholy/ttygif) - Convert terminal recordings to animated gifs
 
+## Demo Tools
+*Tools that can be used to show running software during a presentation* 
+- [Katacoda](https://katacoda.com/) - Create online self-paced learning resources in virtual environment to test software. 
```

Comment:
Can you create a new Learning Platform section, please?
You can put this link into this new section!

Reply:
ok